### PR TITLE
Basic support for c2pa actions

### PR DESCRIPTION
**Pull request by emensch**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [ ] All applicable changes have been documented
- [ ] Any `TO DO` items (or similar) have been entered as GitHub issues and the link to that issue has been included in a comment

### CODE REVIEW

File: packages/c2pa/src/resolvers/editsAndActivity.ts [hunk 1]
```diff
@@ -139,7 +245,7 @@ export const editsAndActivity = createTypedResolvers({
   editsAndActivity: {
     get: (manifest) => getCategorizedActions.bind(null, manifest),
     // @TODO how to handle parameterized calls?
-    getSerializable: async (manifest) => getCategorizedActions(manifest),
+    getSerializable: async (manifest) => getCategorizedActions(manifest) as any,
```
Comment (emensch) [inline packages/c2pa/src/resolvers/editsAndActivity.ts:248]: The serialization type is too restrictive - probably better not to try and enforce anything here. I want to look into this as part of my API exploration

File: packages/c2pa/src/resolvers/editsAndActivity.ts [hunk 2]
```diff
@@ -72,7 +145,32 @@ const processCategories = flow(
...
+
+  if (!actions) {
+    return null;
+  }
+
+  const uniqueActionLabels = actions
+    ?.map(({ action }) => action)
+    .filter(
+      (val, idx, self) =>
+        ACTION_DICTIONARY.hasOwnProperty(val) && self.indexOf(val) === idx,
```
Comment (dkozma) [inline packages/c2pa/src/resolvers/editsAndActivity.ts:161]: What does this part do?
```
self.indexOf(val) === idx
```

**Review by dkozma — APPROVED** [review]
Looks good! Just one small question.

File: packages/c2pa/src/resolvers/editsAndActivity.ts (hunk 2 above)
Comment (emensch) [reply packages/c2pa/src/resolvers/editsAndActivity.ts:161]: De-dupes the array

**Pull request merged**